core/App/screens/ProofRequestAttributeDetails.tsx

```diff
@@ -60,13 +60,13 @@ const ProofRequestAttributeDetails: React.FC<ProofRequestAttributeDetailsProps>
       headerRight: undefined,
       headerLeft: () => (
         <HeaderLeftBack
-          title={t('Global.Back')}
+          title={' '}
           onPress={() => {
             navigation.pop()
           }}
         />
       ),
-      headerBackTitle: t('Global.Back'),
+      headerBackTitleVisible: false,
       headerBackAccessibilityLabel: t('Global.Back'),
```

Comment:
Can remove the a11y label if we're not displaying anything

Reply:
okay got it, I wasn't sure if it was needed for screen readers

Reply:
change completed